docs/pages/release_notes.md

```diff
@@ -20,6 +20,8 @@ This is a {{ site.pmd.release_type }} release.
 ### New and noteworthy
 
 ### Fixed Issues
+* core
+    * [#4248](https://github.com/pmd/pmd/issues/4248): \[core] Can't analyze sources in zip files
```

Comment:
this shouldn't be here for a change in `pmd/7.0.x`

Reply:
just ignore this file "release_notes.md" - this is irrelevant on the pmd/7.0.x branch. It always contains the release notes from master.

The PMD 7 release notes are in the file [7_0_0_release_notes.md](https://github.com/pmd/pmd/blob/pmd/7.0.x/docs/pages/7_0_0_release_notes.md).

We use a different file on pmd/7.0.x in order to avoid merge conflicts.